projects/fsastorefrontlib/src/core/my-account/connectors/sign-documents.adapter.ts

```diff
@@ -0,0 +1,14 @@
+export abstract class SignDocumentsAdapter {
+  /**
```

Comment:
Maybe just to extend existing documentsadapter with signdocuments methods

Reply:
Ok, it does not exist but let's rename this adapter to DocumentAdapter as we have for example PolicyAdapter

Reply:
Naming fixed